app/models/webhooks/slack/base.rb

```diff
@@ -0,0 +1,50 @@
+Webhooks::Slack::Base = Struct.new(:event) do
+
```

Comment:
Totally, I spun this off from loomio into metamaps land first, and then over to here. Absolutely worth noting that @gdpelican wrote a lot of this originally and should get a lot of the credit :)